frontend/public/kubevirt/components/vm.jsx

```diff
@@ -275,7 +275,7 @@ export const VirtualMachinesDetailsPage = props => {
 
   const nicsPage = {
     href: 'nics',
-    name: 'Networks',
+    name: 'NICs',
```

Comment:
I think it should be a full name: `Network Intefaces`. 
Many things are in a full name in tectonic.

Reply:
@andybraren @lizsurette 
I saw https://github.com/kubevirt/web-ui/issues/103 but think that `Network Intefaces` would be  more aligned with other tabs/tectonic. No specific need to use an acronym here IMO.